### DISCUSSION TITLE
How to reuse the old cookie in bot made by puppeteer?

### DISCUSSION THREAD

**Discussion by superdev1218999** [community/community#81822] [Codespaces]
### Select Topic Area

Question

### Body

Nowadays, I'm making the chatbot with puppeteer.
Then I have to login using OTP every time.
It's very inconviente.
So I'm going to reuse the old non-expired cookie to solve this issue.
How can I do it?

**Comment by lightsuperred** [comment]
Hi, I have this solution.
Please refer on https://stackoverflow.com/questions/56514877/how-to-save-cookies-and-load-it-in-another-puppeteer-session.
